### PR TITLE
Merge feat/nextjs-join-us into feat/nextjs

### PR DESCRIPTION
**Pull request by Eddyyyh**

### CODE REVIEW

File: frontend/pages/joinUs.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                          height="250px"
+                          width="250px"
+                        />
+                      </a>
+                    </BootstrapTooltip>
+                  );
+                })}
+              </div>
+            </div>
+            <div className={styles.googleMap}>
```
Comment (joanna209) [inline frontend/pages/joinUs.js:79]: Dimensions of the google map should match the website. Open local instance on one tab and have the wit website on another tab to compare. 

Submitted:
image

What is on the website: 
image

File: frontend/pages/joinUs.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                src="https://maps.google.com/maps?width=100%25&amp;height=600&amp;hl=en&amp;q=UNSW%20Sydney%20High%20St%20Kensington,%20NSW%202052%20Australia+(UNSW%20Sydney)&amp;t=&amp;z=14&amp;ie=UTF8&amp;iwloc=B&amp;output=embed"
+              />
+            </div>
+          </div>
+          {/* End of connect with us section */}
+
+          {/* Start of how to join section */}
+          <h2 className={styles.section}>How to join</h2>
+          <div className={styles.descriptionHowto}>
+            <strong>Interact through social media</strong>
```
Comment (joanna209) [inline frontend/pages/joinUs.js:105]: Same issue as above but with font size. 

Submitted:
image

Website:
image

File: frontend/components/QuotesSlideshow.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+        >
+          {displaySlideInfo(key, quotes.defaultRoleColour)}
+        </div>
+      );
+    }
+    return quoteBlock;
+  };
+
+  const displaySlideInfo = (key, colorClass) => {
+    return (
```
Comment (joanna209) [inline frontend/components/QuotesSlideshow.js:139]: Remove extraneous whitespace for quotes slideshow

image

File: frontend/components/QuotesSlideshow.js [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+      <>
+        <div className={quotes.authorBlock}>
+          <Image
+            className={quotes.authorImage}
+            src={`/../public/portraits/${data[key][2]}`}
```
Comment (joanna209) [inline frontend/components/QuotesSlideshow.js:144]: Fix image error 

> NormalizeError: Requested and resolved page mismatch: /../public/portraits/2022-exec/VivWang.jpg /public/portraits/2022-exec/VivWang.jpg

**Review by joanna209 — CHANGES_REQUESTED** [review]

File: frontend/components/LoadingScreen.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Image from "next/image"
+
+const LoadingScreen = () => {
+  return (
+    <div id="loadingBackground">
+      <Image
+        id="loadingWillow"
+        src={`/walkingWillow.gif`}
```
[the remark this answers is not in the document]
Comment (Eddyyyh) [reply frontend/components/LoadingScreen.js:9]: Done !

File: frontend/components/NewsletterForm.js [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+        Stay up to date WIT us on the latest events and news
+      </p>
+      <form
+        className="newsletter"
+        action={process.env.REACT_APP_MAILCHIMP_URL}
+        target="_blank"
+        method="POST"
+        noValidate
+      >
+        <input
```
[the remark this answers is not in the document]
Comment (Eddyyyh) [reply frontend/components/NewsletterForm.js:42]: Done !

File: frontend/components/QuotesSlideshow.js (hunk 3 above)
Comment (Eddyyyh) [reply frontend/components/QuotesSlideshow.js:139]: done HEHE

**Review by joanna209 — APPROVED** [review]

**Pull request merged**